examples/player/simid_player.js

```diff
@@ -440,11 +468,24 @@ class SimidPlayer {
   
   /** The creative wants to play video. */
   onRequestPlay(incomingMessage) {
-    this.adVideoElement_.play().then(
-      // The play function returns a promise.
-      this.simidProtocol.resolve(incomingMessage),
-      this.simidProtocol.reject(incomingMessage)
-    );
+    
+    if (this.isLinearAd_) {
+      this.adVideoElement_.play()
+      .then(() => this.simidProtocol.resolve(incomingMessage))
+      .catch(() => {
+        errorMessage = {
```

Comment:
`const errorMessage` and below
js is weird, itll work if you never actually declare the variable (by using let, var, or const), but itll put the variable into the global scope by default, which isn't ideal

Reply:
done